src/trace/trace.c

```diff
@@ -137,7 +137,7 @@ static inline bool trace_filter_verbosity(uint32_t lvl, const struct tr_ctx *ctx
 static void emit_recent_entry(struct recent_log_entry *entry)
 {
 	_log_message(trace_log_unfiltered, false, LOG_LEVEL_INFO, _TRACE_INV_CLASS, &dt_tr,
-		     _TRACE_INV_ID, _TRACE_INV_ID, "Suppressed %u similar messages: %pQ",
+		     _TRACE_INV_ID, _TRACE_INV_ID, "Suppressed %u similar messages: %uQ",
 		     entry->trigger_count - CONFIG_TRACE_BURST_COUNT,
 		     entry->entry_id);
```

Comment:
"%p" would have printed a hex pointer, "%u" is going to emit a decimal number.  Seems like we should be casting the argument to "(void *)" vs. changing the format string.

Reply:
sure, I could go with casting as well, I did not attempt to build sof standalone to see how this fits in with what SOF does, my assumption was those were late introductions to the code that worked before..